### ISSUE TITLE
Tab Center ins't properly removed a second time from about:addons

### ISSUE THREAD

**Issue by SoftVision-CiprianMuresan** [bwinton/TabCenter#936]
[Notes]:
- In this state, the buttons from the Tab Center Sidebar no longer work.
- The issue is not reproducible on the custom build we made yesterday.
- The issue is not reproducible on the Test Pilot Dev and Production versions of Tab Center.
- Tab Center is properly removed/disabled after a restart.

[Affected versions]:
- Firefox 51.0.1 and above
- Tab Center 1.29.0 (built on 2/17/2016)

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- The "xpinstall.signatures.required" is set to "false" and latest Tab Center add-on is installed on a new clean profile.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites.
2. Navigate to about:addons->Extensions and remove Tab Center.
3. Click the "Undo" button.
4. Remove or Disable the Tab Center add-on again and observe the browser behavior.

[Expected result]:
- The add-on is removed/disabled and the tabs are moved back on top.

[Actual result]:
- The add-on is removed/disabled, but the tabs remain on the side.

[Additional Notes]:
- Here is a screen recording of the issue: 
![addon cannot be removed a second time from aboutaddons](https://cloud.githubusercontent.com/assets/18304511/23066616/37e1711e-f524-11e6-8ea0-360ece8fddee.gif)

**Comment by ericawright** [comment]
I'm a bit confused about whether this issue is or is not reproducible with the current build?

**Comment by SoftVision-CiprianMuresan** [comment]
The issue is reproducible on the custom build I made today (2/20/2017). 
The issue appeared on 2/17/2017 and it is not reproducible on the on the custom I made the day before that (2/16/2017). (I can provide the xpi's for the builds in question, if you want them)

**Comment by ericawright** [comment]
is this still happening?

**Comment by ericawright** [comment]
Yeah - I believe it was the side error that was fixed in that PR. The error that was happening when the tour attempted to trigger but the tabs were on the side.
